src/pages/AdminManageAccess.js

```diff
@@ -177,8 +162,23 @@ class AdminManageAccess extends Component {
   };
 
   render() {
-    const { searchDarText, currentPage, limit } = this.state;
+    const { showModal, searchDarText, currentPage, limit, darDetails, researcher } = this.state;
     const pageCount = Math.ceil((this.state.darElectionList.filter(this.searchTable(searchDarText)).filter(row => !row.isCanceled).length).toFixed(1) / limit);
+    const handleCloseModal = () => {
+      this.setState({ showModal: false });
+    };
+    const openSummaryModal = async (dar) => {
+      let darDetails = await DAR.getDarModalSummary(dar.dataRequestId);
```

Comment:
Should use `DAR.describeDar()` instead - that will give you all of data necessary. The modal summary API should be avoided.

Reply:
so DAR.describeDar() has a list of dataset ids, but it does not have a list of datasets or a list of dataset names, like dar.getDarModalSummary has. We need the dataset names for the modal, so if we do want to switch describeDar then we would need an extra step to get dataset by ID... I am thinking this would work  well if we move the getDatasetNames function there to the DarModal

Reply:
or if we want to change the whole page over to the V2 API call then we wouldn't even need an extra API call in openSummary cause we'd already have all the info, but that would be a big change, not sure what you think?

Reply:
You can get the datasets from the datasets API.